azurefunctions-extensions-bindings-servicebus/azurefunctions/extensions/bindings/servicebus/serviceBusMessageActions.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) .NET Foundation. All rights reserved.
+# Licensed under the MIT License.
+
+from typing import Optional
+
+from azurefunctions.extensions.base import GrpcClientType
+from google.protobuf.wrappers_pb2 import StringValue
+
+from ..protos.settlement_pb2 import (
+    AbandonRequest,
+    CompleteRequest,
+    DeadletterRequest,
+    DeferRequest,
+    ReleaseSessionRequest,
+    RenewMessageLockRequest,
+    RenewSessionLockRequest,
+    SetSessionStateRequest,
+)
+from ..protos.settlement_pb2_grpc import SettlementStub
+
+from .grpcClient import GrpcClientFactory
+from .grpc_utils import build_grpc_uri
+
+
+class ServiceBusMessageActions(GrpcClientType):
+    """
+    ServiceBusMessageActions class.
+    Provides async methods for message settlement over gRPC.
+    Implements a singleton pattern.
+    """
+
+    _instance: Optional["ServiceBusMessageActions"] = None
+
+    def __init__(self) -> None:
+        self._uri, self._grpc_max_message_length = build_grpc_uri()
+
+        self._client: SettlementStub = GrpcClientFactory.create_client(
+            service_stub=SettlementStub,
+            address=self._uri,
+            grpc_max_message_length=self._grpc_max_message_length,
+            secure=False,
```

Comment:
Why is this set to False? This should depend on whether secure is needed or not right?